docs/UserGuide.md

```diff
@@ -239,10 +239,16 @@ Delete existing event(s).
 * `deleteevent 2`
 
 ### Show all events : `showevents`
+Shows all the upcoming events in amigos
 
-Shows all events stored in Amigos. Switches GUI to the events tab.
+**Format**: `showevents [-a]`
 
-**Format**: `showevents`
+* Switches GUI to the events tab
+* if the `-a` flag is provided it will show all events in Amigos 
+
+**Examples**:
+* `showevents`
+* `showevents -a`
```

Comment:
I think there can be greater clarity here - what is the difference between showing all events and the one without the -a flag (i.e. only show stuff from today onwards???)